src/systems/ec/implementations/react/components/gallery/src/GalleryItem.jsx

```diff
@@ -55,6 +77,21 @@ GalleryItem.propTypes = {
   item: PropTypes.shape({
     src: PropTypes.string,
     alt: PropTypes.string,
+    image: PropTypes.string,
```

Comment:
`image` is used for the video, while `src` is used for the image... this is very confusing
can we provide a better alternative?

Reply:
we can't change the attributes already used for image, and for video I used the same as in media container component

Reply:
`src` could be used as a legacy
then we add `video` and `image`, which contain the sub-properties 

```
src: PropTypes.string, // Legacy, please use "image.src" instead
video: {
  tracks: ...,
  sources: ...,
  poster: ...,
},
image: {
  src: PropTypes.string,
  srcset: PropTypes.string, 
  whatever...
}
```